Cubical/Foundations/Equiv.agda

```diff
@@ -145,15 +145,24 @@ Lift≃Lift e .snd .equiv-proof b .snd (a , p) i .snd j .lower =
 isContr→Equiv : isContr A → isContr B → A ≃ B
 isContr→Equiv Actr Bctr = isoToEquiv (isContr→Iso Actr Bctr)
 
-isPropEquiv→Equiv : (Aprop : isProp A) (Bprop : isProp B) (f : A → B) (g : B → A) → A ≃ B
-isPropEquiv→Equiv Aprop Bprop f g = f , hf
+propBiimpl→Equiv : (Aprop : isProp A) (Bprop : isProp B) (f : A → B) (g : B → A) → A ≃ B
```

Comment:
`<->` exists and is called `_⇔_` in `Foundations/Logic`, let me double check that there wasn't a reason not to directly move it, given that `⇔toPath` also lives there.

Reply:
I see, I keep forgetting about the Foundations.Logic file... I think my conclusion last time I looked at it was that it contained a few useful things, but lots of stuff can be moved/removed, and that some proofs could be simplified. I'll make an issue about it so that I don't forget about it